.github/workflows/create-urgent-hotfix-img.yml

```
@@ -0,0 +1,301 @@
+              esac
+            else
+              echo "❌ Invalid format: $ITEM"; exit 1
+            fi
+          done
+
+          printf '%s\n' "${VALID_COMPONENTS[@]}" | \
+            jq -R -s -c 'split("\n") | map(select(length > 0)) | map(split(":") | {component: .[0], version: .[1]})' | \
+            tee >(echo "matrix=$(cat)" >> "$GITHUB_OUTPUT")
+
+  build-images:
+    name: Build `${{ matrix.component }}:${{ matrix.version }}` Image
+    runs-on: gcp-perf-core-8-default
+    needs: validate-inputs
+    timeout-minutes: 45
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    outputs:
+      short-sha: ${{ steps.get-sha.outputs.short-sha }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include: ${{ fromJson(needs.validate-inputs.outputs.components-matrix) }}
+    steps:
+      - name: Checkout and resolve ref
+        id: resolve
+        uses: actions/checkout@v5
+        with:
+          token: ${{ github.token }}
+          ref: ${{ needs.validate-inputs.outputs.commit || needs.validate-inputs.outputs.branch }}
+      
+      - name: Get resolved SHA
+        id: get-sha
+        run: |
+          # If commit was provided, use it; otherwise resolve from checked out branch
+          COMMIT="${{ needs.validate-inputs.outputs.commit }}"
+          if [[ -n "$COMMIT" ]]; then
+            FULL_SHA="$COMMIT"
+            SHORT_SHA=$(git rev-parse --short "$COMMIT")
+          else
+            FULL_SHA=$(git rev-parse HEAD)
+            SHORT_SHA=$(git rev-parse --short HEAD)
+          fi
+          
+          echo "✅ Building from commit: $FULL_SHA (short: $SHORT_SHA)"
+          echo "full-sha=$FULL_SHA" >> "$GITHUB_OUTPUT"
+          echo "short-sha=$SHORT_SHA" >> "$GITHUB_OUTPUT"
+          
+      - name: Import Secrets
+        id: secrets
+        uses: hashicorp/vault-action@v3.4.0
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secrets: |
+            secret/data/github.com/organizations/camunda NEXUS_PSW | ci-account-password;
+            secret/data/github.com/organizations/camunda NEXUS_USR | ci-account-username;
+
+      - name: Log into Harbor
+        uses: docker/login-action@v3
+        with:
+            registry: registry.camunda.cloud
+            username: ${{ steps.secrets.outputs.ci-account-username }}
+            password: ${{ steps.secrets.outputs.ci-account-password }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '21'
+
+      - name: Build Maven artifacts
+        run: |
+          echo "🔨 Building Maven artifacts for ${{ matrix.component }}..."
+
+          case "${{ matrix.component }}" in
+            "zeebe"|"camunda"|"operate"|"tasklist")
```

alphabetical recommended here as well for readability:

```suggestion
            "camunda"|"operate"|"tasklist"|"zeebe")
```